### DISCUSSION TITLE
Signal can not create case automatically?

### DISCUSSION THREAD

**Discussion by tiankai0114** [Netflix/dispatch#5181] [Q&A]
Hi Team,

I added signal definition and checkbox create case, however when I use api to create a single instance, response is 200 ok, however only create a signal on Dispatch, did not create case, did everyone knows what happened? from web logs, shows only create signal api, did not have trigger any api about case.

I use the latest code for deployment, did I do something wrong? Has anyone else had this problem?

**Comment by mvilanova** [comment]
Hi, the signal instance gets created when you hit the API, but in order for the instance to be processed you need to run `dispatch signals process`. We also have `dispatch signals consume` if you want to consume signals from an AWS SQS queue if you want. Hope this helps.
